test/controllers/links_controller_test.rb

```diff
@@ -5,14 +5,12 @@ class LinksControllerTest < ActionDispatch::IntegrationTest
   
```

Comment:
Getting an error on this test. Not sure how it related to this PR anyway though?

<img width="731" alt="Screenshot 2022-03-28 at 17 45 14" src="https://user-images.githubusercontent.com/40244233/160447043-5d690d66-63de-48aa-87ea-7a28aacbde37.png">

```
Error:
LinksControllerTest#test_should_update_a_link:
NoMethodError: undefined method `name' for #<Link:0x000000011ebf2e70>
    test/controllers/links_controller_test.rb:10:in `block in <class:LinksControllerTest>'
```